lib/generators/decidim/templates/initializer.rb

```diff
@@ -7,7 +7,7 @@
   config.authorization_handlers = [ExampleAuthorizationHandler]
 
   # Uncomment this lines to set your preferred locales
-  # config.available_locales = %i{en ca es}
```

Comment:
It is. But we are enforcing the style I switched to via [rubocop](https://github.com/decidim/decidim/blob/fa4755756c85041671bf992a3541d7f1ebb74525/.rubocop.yml#L939-L944).